docs/dev/core/internal/payload.md

```diff
@@ -75,7 +75,28 @@ A [Timing distribution](../../../user/metrics/timing_distribution.md) is represe
 | Field name | Type | Description |
 |---|---|---|
 | `sum` | Integer | The sum of all recorded values. |
-| `values` | Map&lt;String, Integer&gt; | The values in each bucket. The key is the minimum value for the range of that bucket. Buckets with no values are not reported. |
+| `values` | Map&lt;String, Integer&gt; | The values in each bucket. The key is the minimum value for the range of that bucket. All contiguous buckets between the minimum non-zero bucket and the maximum non-zero bucket are included. |
```

Comment:
So, all of this has been in a bit of flux so you're right to call for some clarification.

Basically it's this:

- For functional distributions, the range is from the first bucket with a non-zero value, to the bucket after the last bucket with a non-zero value -- since you need to have one extra bucket so you know where the last bucket ends.  You don't want to start at the first bucket, since for these functional histograms it can be a lot of buckets.

- For custom distributions, the range is from the first bucket regardless of its value (defined by rangeMin) to the bucket after the last bucket with the non-zero value.

I'll try to come up with a way to explain this more clearly and update this PR.